redis/reply_test.go

```diff
@@ -333,3 +333,40 @@ func ExampleString() {
 	// Output:
 	// "world"
 }
+
+func ExampleBytesMap() {
```

Comment:
Could you clarify why would you not just use `Values(...)` or `ScanStruct` a for this?